js/modules/modules.go

```diff
@@ -69,6 +70,11 @@ type HasModuleInstancePerVU interface {
 	NewModuleInstancePerVU() interface{}
 }
 
+// IsModuleV2 ... TODO better name
+type IsModuleV2 interface { // TODO rename?
```

Comment:
Just `Module`? Or do we want to keep supporting `common.Bind()`?

Reply:
I would prefer to keep supporting common.BInd for at least 1 more release ... especially as otherwise this will mean we will need to rewrite all modules ... now, which will probably be a bit too much work for the remaining ... 2 weeks ? or something like that